<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Window</class>
 <widget class="QDialog" name="Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>717</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btnConTracker">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Connect Tracker</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnGetMemList">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>370</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>get MemList</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnConClients">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>370</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Connect Clients</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnStart">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>370</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Start Sending</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="chat">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>701</width>
     <height>261</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnStop">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>370</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop Sending</string>
   </property>
  </widget>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>720</width>
     <height>400</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>background</zorder>
  <zorder>btnConTracker</zorder>
  <zorder>btnGetMemList</zorder>
  <zorder>btnConClients</zorder>
  <zorder>btnStart</zorder>
  <zorder>chat</zorder>
  <zorder>btnStop</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
